mkdocs-base.yml

```diff
@@ -143,7 +143,7 @@ nav:
   - cluster-management.md
   - db_engine_config.md
   - db_provision.md
-
+  - gke.md
```

Comment:
I think we need to have it under `Configure -> Set up Kubernetes clusters`. @PeterSzcz what do you think?

Reply:
This is just the topic content. We will work on the structure after all the topics are completed.